buildout.d/versions.cfg

```diff
@@ -2,6 +2,7 @@
 setuptools = 8.3
 zc.buildout = 2.2.5
 zc.recipe.egg = 2.0.1
+plone.app.widgets = 1.8.0
```

Comment:
@prakharjoshi why did we have to pin p.a.widgets? That package should not be included at all.